halo2-base/src/lib.rs

```diff
@@ -432,8 +432,9 @@ impl<F: ScalarField> Context<F> {
     /// The `MockProver` will print out the row, column where it fails, so it serves as a debugging "break point"
     /// so you can add to your code to search for where the actual constraint failure occurs.
     pub fn debug_assert_false(&mut self) {
-        let three = self.load_witness(F::from(3));
-        let four = self.load_witness(F::from(4));
-        self.constrain_equal(&three, &four);
+        use rand_chacha::rand_core::OsRng;
```

Comment:
oops change this back; `load_constant` turns out not to be useful for debugging

Reply:
OsRng is ok to keep

Reply:
Fixed.